commit f6adcca02ca26577692a1f2ee892c134b076b339
Author: Vishal Verma <vishal.l.verma@intel.com>
Date:   Thu Feb 8 14:24:05 2018 -0700

    libnvdimm, testing: update the default smart ctrl_temperature
    
    The default value for smart ctrl_temperature was the same as the
    threshold for ctrl_temperature. As a result, any arbitrary smart
    injection to the nfit_test dimm could cause this alarm to trigger
    and cause an acpi notification. Drop the default value to below the
    threshold, so that unrelated injections don't trigger notifications.
    
    Cc: Dan Williams <dan.j.williams@intel.com>
    Signed-off-by: Vishal Verma <vishal.l.verma@intel.com>
    Signed-off-by: Dan Williams <dan.j.williams@intel.com>

diff --git a/tools/testing/nvdimm/test/nfit.c b/tools/testing/nvdimm/test/nfit.c
index e83c24625fe4..cb166be4918d 100644
--- a/tools/testing/nvdimm/test/nfit.c
+++ b/tools/testing/nvdimm/test/nfit.c
@@ -1259,7 +1259,7 @@ static void smart_init(struct nfit_test *t)
 			| ND_INTEL_SMART_MTEMP_VALID,
 		.health = ND_INTEL_SMART_NON_CRITICAL_HEALTH,
 		.media_temperature = 23 * 16,
-		.ctrl_temperature = 30 * 16,
+		.ctrl_temperature = 25 * 16,
 		.pmic_temperature = 40 * 16,
 		.spares = 75,
 		.alarm_flags = ND_INTEL_SMART_SPARE_TRIP